addons/l10n_hu_edi/i18n/hu.po

```diff
@@ -54,19 +54,13 @@ msgstr ""
 
 #. module: l10n_hu_edi
 #: model_terms:ir.ui.view,arch_db:l10n_hu_edi.account_move_send_form
-msgid ""
```

Comment:
It seems to me that this is unnecessary diff (maybe due to the fact that your .po tool works differently from Python's polib?)

I think we want to keep everything as close as possible to the format generated by polib. This is because the translations are re-generated every major version, so if we don't keep things in polib format now, it will create a diff in the next major version, which will cause annoying fw-port conflicts.